generator/src/main/java/com/scottlogic/deg/generator/guice/MonitorProvider.java

```diff
@@ -24,6 +24,9 @@
 
     @Override
     public ReductiveDataGeneratorMonitor get() {
+        if (commandLine.getOutputDestination().getDestination().equals("STREAM")) {
```

Comment:
Use the field Destinations.STREAM explicitly. This keeps the reference tight, so if someone changes the name either here or there, the other is updated.

Reply:
Definitely shouldn't be doing string checking when you have an enum type.

Reply:
Whether streaming is enabled or not shouldn't preclude the ability to have velocity reporting. The velocity reports, etc. should be emitted to STDERR instead.